### DISCUSSION TITLE
3rd Floor Doors not showing with Perfect Vision Enables

### DISCUSSION THREAD

**Discussion by AEYST** [dev7355608/perfect-vision#317] [General]
Foundry VTT V10, latest release issue.  I noticed this a couple day ago as well.  I use levels, lib wrapper, Tidy UI, Wall Height, and Perfect Vision.  When I enable perfect vision on a scene for a distance other than infinity, my horizontal doors on the 3rd level are not visible.

**Comment by dev7355608** [comment]
Hard to tell that the issue is with that little information. Make sure you've actually got the latest version of Wall Height (4.9), Levels (3.5.8), and Perfect Vision (4.1.4). There have been some changes recently regarding visibility that require all three modules to be up to date.

**Reply by AEYST** [reply]
All modules are running the current version.  If there’s additional information needed I be happy to provide it.  I’m using D&D5e, if that helps.

**Reply by dev7355608** [reply]
Try the latest Levels version that was just released and report back whether that fixed the issue. If it didn't fix it, then I need the exported scene to be able to reproduce this issue.

**Reply by AEYST** [reply]
New levels version fixed the issue.  Thank you.
